gitflow.html

```diff
@@ -52,31 +52,171 @@
 ## A branching model for Git
         
 .right[![Right-aligned image](https://avatars2.githubusercontent.com/u/52010894?s=200&v=4)]
-        
+
 ---
-               
-## The Branches
+
+## What Is GitFlow??
+
+--
+GitFlow is a branching model for Git, created by Vincent Driessen. It has attracted a lot of attention because it is very well suited to collaboration and scaling the development team. We think this model is very well suited with our current release process at ESDC.  
+
+--
+### Why Is GitFlow??  
+
+--
+Key Benefits
+
+--
+- Separation of concepts - each branch has clear purpose and flow how to work with it  
+
+--
+- Confidence - at any given point of time master and release branches are stable and ready for production  
+
+--
+- Isolation - you develop independent and isolated feature, so you can focus on problem  
```

Comment:
Grammar, but a bit of my personal preference for wording.
```suggestion
- Isolation - you develop independent and isolated features, so you can focus on one problem at a time
```